google/cloud/aiplatform/experimental/vertex_model/serializers/dataloaders.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
```

Comment:
Possibly rename this file to pytorch.py.

Reply:
will we be using other pytorch data types?